src/Authentication.Abstractions/Interfaces/IAzureEnvironment.cs

```diff
@@ -136,5 +136,15 @@ public interface IAzureEnvironment : IExtensibleModel
         /// The domain name suffix for Azure Container Registry
         /// </summary>
         string ContainerRegistryEndpointSuffix { get; set; }
+
+        /// <summary>
+        /// The MSGraph endpoint
+        /// </summary>
+        string MicrosoftGraphUrl { get; set; }
+
+        /// <summary>
+        /// The MSGraph Resource Url endpoint
+        /// </summary>
+        string MicrosoftGraphEndpointResourceId { get; set; }
```

Comment:
When adding new endpoints (resource IDs), we typically add them as extended properties of IAzureEnvironment, so that the interface doesn't need to change.
You can refer to this example: https://github.com/Azure/azure-powershell-common/pull/227/files

Reply:
Resolved